src/api/hooks/Utils.tsx

```diff
@@ -121,3 +122,16 @@ export const usePersistedState = <T,>(key: string, defaultValue: T, duration = 3
 
   return [state, setState] as const;
 };
+
+export const useGoogleAnalytics = () => {
+  const event = useCallback((action: string, category: string, label: string) => {
```

Comment:
Legge på JsDoc her med forklaring på hva de forskjellige skal inneholde/bety?

Reply:
Noe grunn til at du vil ha det som en hook forresten, og ikke bare som en util-funksjon som kan importeres uten å kjøre `const { event } = useGoogleAnalytics();` ?

Reply:
Will do! Tenkte det var best practice og mest semantisk i React. Sørger vel også for at ikke funksjonen oppdateres for hver gang, men at den samme brukes overalt. Gjør forsåvidt at vi også kan legge til flere GA-funksjoner i hooken uten å måtte importere en ny funksjon

Reply:
Enig der 👍 Hvis jeg skal være veldig pirkete så hadde det vært nice om rekkefølgen på parametrene action og category var motsatt, fordi category har actions som har labels. Men ikke noe veldig big deal

Reply:
category > actions >label ? Navngivingen til google er ikke helt på topp 👍

Reply:
Ikke heeelt selvforklarende nei: https://support.google.com/analytics/answer/1033068#Anatomy

![Screenshot_20210723-144644_Chrome](https://user-images.githubusercontent.com/31009729/126783586-28ad9410-e2da-4e0d-b21f-34167a7abedc.jpg)